src/utils/sprint-utils.ts

```diff
@@ -109,6 +109,40 @@ export const getSeveraUserId = (user: User): string => {
   return user?.attributes?.severaUserId ?? config.user.testUserSeveraId ?? "";
 };
 
+/**
+ * Get total work hours for each phase 
+ * 
+ * @param workHours with type WorkHours[]
+ * @param phase with type Phase
+ * @param userId with type string
+ * 
+ * @returns total work hours for each phase according to the userId
+ */
+const totalWorkHours = (workHours: WorkHours[], phase: Phase, userId: string) => {
+  return workHours
+    .filter(workHour => workHour.phase?.severaPhaseId === phase.severaPhaseId && workHour.user?.severaUserId === userId)
```

Comment:
I think this could be simplified, or at least made clearer by having the phaseId comparison and userId comparison as named variables?